Formula/wildfly-as.rb

```diff
@@ -13,7 +13,7 @@ def install
   end
 
   def caveats; <<-EOS.undent
-    The home of WildFly Application Server 10 is:
+    The home of WildFly Application Server 10.1 is:
```

Comment:
No, if you feel like its better to not do this change, let me know, i ll revert it.